### PR TITLE
SWORD2: Fix support for Korean translation

### PR DESCRIPTION
**Pull request by british-choi**
### Broken Sword II: The Smoking Mirror (Windows/Korean)

This supports Korean fan translations.

Korean FAN translation data is available at the link below.

[https://github.com/british-choi/ScummVM-Kor-Trs](https://github.com/british-choi/ScummVM-Kor-Trs)

image

### CODE REVIEW

File: engines/sword2/resman.cpp [hunk 1]
```diff
@@ -230,6 +230,9 @@ bool ResourceManager::init() {
 				return false;
 			}
 
+			// Use korean.clu instead of TEXT.CLU
+			if (_vm->_isKorTrs && !scumm_stricmp(_resFiles[i].fileName, "TEXT.CLU"))
+				strcpy_s(_resFiles[i].fileName, 20, "korean.clu");
```
Comment (AndywinXp) [inline engines/sword2/resman.cpp:235]: Please use `Common::strcpy_s(_resFiles[i].fileName, 20, "korean.clu")` so that Linux and Mac can build 🙂

**Review by AndywinXp — CHANGES_REQUESTED** [review]
Hi! Thanks for issuing a PR yourself for these changes, I appreciate it! 😄 
This looks good apart from one simple changes which has to be done in order for this to build on all systems.

File: engines/sword2/resman.cpp (hunk 1 above)
Comment (british-choi) [reply engines/sword2/resman.cpp]: Thanks, I fixed it

**Review by AndywinXp — APPROVED** [review]

**Comment by AndywinXp** [comment]
Thank you! 😄 Merging!

**Pull request merged**